### PR TITLE
Update Photourl for Suwon

### PR DESCRIPTION
**Pull request by hskang9**
Upload picture for free code camp suwon 😆

### CODE REVIEW

**Comment by ginoskotheon** [comment]
@hskang9 I'll get on this when I get off of work.

File: assets/json/campsitesfinal.json [hunk 1]
```diff
@@ -8650,10 +8650,10 @@
   {
     "url": "https://www.facebook.com/groups/free.code.camp.suwon",
     "city": "Suwon",
-    "state": "",
+    "state": "Gyeonggi-do",
```
Comment (hallaathrad) [inline assets/json/campsitesfinal.json]: :+1:

File: assets/json/campsitesfinal.json [hunk 2]
```diff
@@ -8650,10 +8650,10 @@
...
     "country": "South Korea",
     "coordinates": "33.429441, 126.267859",
-    "photoUrl": ""
+    "photoUrl": "https://user-images.githubusercontent.com/12888144/27015187-38d3456a-4f43-11e7-83d3-1bdc810def3c.jpeg"
```
Comment (hallaathrad) [inline assets/json/campsitesfinal.json]: Not to nitpick, but it'd be preferable if the image was hosted on Facebook (your group's album, maybe?)

Comment (hskang9) [inline assets/json/campsitesfinal.json]: I got it

**Comment by raisedadead** [comment]
Hi @hskang9 can you make the necessary changes?

**Comment by hskang9** [comment]
Sure thing!

**Comment by johnkennedy9147** [comment]
@raisedadead I think this can be merged...

**Comment by hallaathrad** [comment]
Thanks for the nudge, @raisedadead 
The notification for this went under my radar.

**Pull request merged**